resources/idlharness.js

```diff
@@ -415,14 +417,14 @@ IdlArray.prototype.is_json_type = function(type)
                    if (thing.has_to_json_regular_operation()) { return true; }
                    var mixins = this.implements[thing.name];
                    if (mixins) {
-                       mixins = mixins.map(id => {
+                       mixins = mixins.map(function(id) {
                            var mixin = this.members[id];
                            if (!mixin) {
                                throw new Error("Interface " + id + " not found (implemented by " + thing.name + ")");
                            }
                            return mixin;
                        });
-                       if (mixins.some(m => m.has_to_json_regular_operation())) { return true; }
+                       if (mixins.some(function(m) { m.has_to_json_regular_operation() } )) { return true; }
```

Comment:
The implicit return from the arrow needs to be an explicit `return ...` in the function expression.